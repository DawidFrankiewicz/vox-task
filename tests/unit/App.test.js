import { mount } from '@vue/test-utils';

const factory = async () => {
    const component = await import('@/App.vue');

    return mount(component.default, {
        shallow: true,
    });
};

test('Should render correctly', async () => {
    const wrapper = await factory();

    // Check if component matches snapshot (HTML structure)
    // https://vitest.dev/guide/snapshot#updating-snapshots
    expect(wrapper.html()).toMatchSnapshot();
});
