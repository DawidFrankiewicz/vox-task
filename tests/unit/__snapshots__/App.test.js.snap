// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Should render correctly 1`] = `
"<div data-v-7a7a37b1=""><a data-v-7a7a37b1="" href="https://vitejs.dev" target="_blank"><img data-v-7a7a37b1="" src="/vite.svg" class="logo" alt="Vite logo"></a><a data-v-7a7a37b1="" href="https://vuejs.org/" target="_blank"><img data-v-7a7a37b1="" src="/src/assets/vue.svg" class="logo vue" alt="Vue logo"></a></div>
<hello-world-stub data-v-7a7a37b1="" msg="Vite + Vue"></hello-world-stub>"
`;
